index.html

```diff
@@ -5,9 +5,158 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+ <link rel="stylesheet" href="/style.css">
+ <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=lato">
+ <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+ <script src="https://kit.fontawesome.com/7967e81f77.js" crossorigin="anonymous"></script>
   <title>Document</title>
 </head>
 <body>
```

Comment:
- [ ] There is some horizontal scroll on the page which would be great to fix, so please work on this issue.